crates/txtx-core/src/runbook/execution_context.rs

```diff
@@ -157,230 +202,442 @@ impl RunbookExecutionContext {
         let (tx, _rx) = unbounded();
 
         for construct_did in ordered_constructs.into_iter() {
-            // Is a command being considered? (vs signer, env, etc)
-            let Some(command_instance) = self.commands_instances.get(&construct_did) else {
-                continue;
+            if let Some(_) = self.commands_instances.get(&construct_did) {
+                match self
+                    .simulate_command_instance(
+                        &construct_did,
+                        &mut pass_result,
+                        &mut unexecutable_nodes,
+                        runtime_context,
+                        workspace_context,
+                        supervision_context,
+                        constructs_dids_frontier,
+                        &tx,
+                    )
+                    .await
+                {
+                    LoopEvaluationResult::Continue => continue,
+                    LoopEvaluationResult::Bail => return pass_result,
+                };
             };
+            // if let Some(_) = self.embedded_runbooks.get(&construct_did) {
```

Comment:
We need ti figure out simulation of runbooks that are embedding another runbook. Planner task [here](https://teams.microsoft.com/l/entity/com.microsoft.teamspace.tab.planner/mytasks?tenantId=0f1e1f86-ff36-4233-ba7c-e2d48a07bb41&webUrl=https%3A%2F%2Ftasks.teams.microsoft.com%2Fteamsui%2FpersonalApp%2Falltasklists&context=%7B%22subEntityId%22%3A%22%2Fv1%2Fpremiumplan%2Fc7a7d962-0836-4ed0-8a67-774535750432%2Forg%2Fdeb736b0-d91b-ef11-9f85-000d3a537223%2Fview%2Fboard%2Ftask%2F1953FD95-C015-4A16-8AA8-81FD7FF49A1A%22%7D)